apps/automation/automation-v2/src/tasks/check-signers-eth-balance.ts

```diff
@@ -22,12 +22,13 @@ export interface SignerWithPrevBalance {
 
 export async function checkSignersEthBalance(ctx: TaskContext, params: Params): Promise<TaskResult> {
     const signers = params.signersWithPrevBalance;
```

Comment:
Origami has a faucet set up, rather than just an alert like this. If the number of tasks grow, it could be worth doing the same.

Idea is you have one EOA which is the faucet and funded (and an alert checks this balance), and then a task which auto allocates funds to all the individual task EOAs from that faucet.